classic-commerce.php

```diff
@@ -16,30 +16,52 @@
 	exit; // Exit if accessed directly.
 }
 
-// Load the Update Client to manage Classic Commerce updates
-include_once dirname( __FILE__ ) . '/includes/class-wc-update-client.php';
-
```

Comment:
Need to add

```
if ( ! function_exists( 'is_plugin_active' ) ) {
	include_once ABSPATH . 'wp-admin/includes/plugin.php';
}
```
Fatal error: Uncaught Error: Call to undefined function is_plugin_active()